src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/UserPrincipalAuditorAware.kt

```diff
@@ -3,12 +3,14 @@ package uk.gov.justice.digital.hmpps.educationandworkplanapi.app.database.jpa
 import org.springframework.data.domain.AuditorAware
 import org.springframework.security.core.context.SecurityContextHolder
 import org.springframework.stereotype.Component
-import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.config.AuthAwareTokenConverter
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.app.config.AuthAwareTokenConverter.Companion.SYSTEM_USER
 import java.util.Optional
 
 @Component
 class UserPrincipalAuditorAware : AuditorAware<String> {
   override fun getCurrentAuditor(): Optional<String> = Optional.of(
-    SecurityContextHolder.getContext()?.authentication?.name ?: AuthAwareTokenConverter.SYSTEM,
+    SecurityContextHolder.getContext()?.authentication?.principal?.let {
```

Comment:
This was the core fix - we need to be using the principal (which is a string, as set in `AuthAwareTokenConverter` above)
Using `name` as before effectively pulls out the JWT `sub` claim, which is not what we want